enable/tools/apptools/__init__.py

```diff
@@ -0,0 +1,10 @@
+import warnings
+
+warnings.warn(
+    ("apptools.undo is deprecated and will be removed in a future release. The"
+     " functionality is now available via pyface.undo. As a result,"
+     " enable.tools.apptools has been deprecated in favor of"
+     " enable.tools.pyface."),
+    DeprecationWarning,
+    stacklevel=2
+)
```

Comment:
This file needs a copyright header

Reply:
I added the same one used in the `enable/tools/apptools/api.py`.  I think these will likely need to be changed once we start using https://github.com/enthought/ets-copyright-checker